### ISSUE TITLE
L1 loss value of AALBERT pre-training task

### ISSUE THREAD

**Issue by kenjewu** [andi611/Self-Supervised-Speech-Pretraining-and-Representation-Learning#42]
Can anyone tell me what is the approximate value of L1 Loss in the training/validation set after 500K iterations are completed during the pre-training task of AALBERT?

**Comment by andi611** [comment]
The value of L1 Loss hardly depends on how the feature is preprocessed. 
That is, values are different for different acoustic features. 
If you want to check whether the reconstruction is good, you can view the plotted images in tensorboard.
